### PR TITLE
Added admin installation steps.

### PR DESCRIPTION
**Pull request by justinegeffen**
Added installation steps.

### CODE REVIEW

**Review by cpoile — APPROVED** [review]
I'm actually not sure how to install it from the marketplace. I'm relying on @itao for an accuracy review here. :)

File: docs/administrator-guide/installation-and-activation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Installing Channel Export for Self-Managed Deployments
+
+Mattermost Incident Management is available in the Plugin Marketplace. You can download and install the plugin from Mattermost.
+
+1. Open **System Console > Plugin Management**.
```
Comment (itao) [inline docs/administrator-guide/installation-and-activation.md:5]: We should clarify that this step to enable the plugin comes after already having installed the plugin from the marketplace. I was confused reading this as the first step.

File: docs/administrator-guide/installation-and-activation.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+2. Search for **Channel Export** using the search bar or scroll through the list manually.
+3. Select **Install**.
+4. Next, select **Configure**.
+5. Select **true** to enable the plugin.
+6. Select **Save**.
+
+When you open the Main Menu, **Playbooks & Incidents** is available as a menu item.
```
Comment (itao) [inline docs/administrator-guide/installation-and-activation.md:12]: That menu item actually has nothing to do with the channel export plugin being installed, we should remove this.

**Review by itao — CHANGES_REQUESTED** [review]

File: docs/administrator-guide/installation-and-activation.md (hunk 1 above)
Comment (justinegeffen) [reply docs/administrator-guide/installation-and-activation.md:5]: Good point, thanks.

Comment (justinegeffen) [reply docs/administrator-guide/installation-and-activation.md:5]: @itao, confirming that this has been addressed.

**Pull request merged**